### DISCUSSION TITLE
How to rename a property colliding with other values

### DISCUSSION THREAD

**Discussion by jsprog** [slint-ui/slint#5530] [Q&A] [answered]
I'm having an in-out property named `center` and I'm planning to access it from rust side with both `set_center` and `get_center`.
Unfortunately, the name is colliding with the `center` value used for alignment and the project is failing to build.

Is it possible to rename the property `center` (e.g: `center_prop`), but keeping the name `set_center` and `set_center` for `rust`

```slint
slint::slint! {
    import { Button, SpinBox } from "std-widgets.slint";

    export component MainWindow inherits Window {
        width: 300px;
        height: 200px;

        in-out property <{x: int, y: int}> center: { x: 0, y: 0 };

        VerticalLayout {
            HorizontalLayout {
                padding: 5px; spacing: 5px;
                
                Text { text: "zoom:"; vertical-alignment: center; } // <- This won't work due to existence of a property with similar name
                SpinBox {}
            }
        }
    }
}
```

**Comment by ogoffart** [comment, marked as answer]
You can qualify the enum value with the enum name.

For example 
`vertical-alignment: TextVerticalAlignment.center;`

I wonder if there could be an easier diagnostic ( see https://github.com/slint-ui/slint/issues/4397 )
